docs/diagrams/blob-transfer.puml

```diff
@@ -0,0 +1,35 @@
+'
+'  Copyright (c) 2022 Microsoft Corporation
+'
+'  This program and the accompanying materials are made available under the
+'  terms of the Apache License, Version 2.0 which is available at
+'  https://www.apache.org/licenses/LICENSE-2.0
+'
+'  SPDX-License-Identifier: Apache-2.0
+'
+'  Contributors:
+'       Microsoft Corporation - Initial document
+'
+'
+
+@startuml
+autonumber "<b>(0)"
+Client -> Consumer: Call DM API: POST /transferprocess \n - managedResources=true\n - Storage account name
+Consumer -> "Consumer Vault": Get dest Storage Account key
+Consumer -> "Consumer Storage Account": Create container
+Consumer -> Provider: IDS message providing: \n - Account name \n - Container name \n - Write-only dest SAS token
```

Comment:
Does the provider also need to know which data is requested from the provider?

Reply:
Yes, the Consumer specifies the asset id to the Provider. I added it in the doc. Thanks!